src/agent/onefuzz/src/process.rs

```diff
@@ -126,3 +131,76 @@ pub async fn run_cmd<S: ::std::hash::BuildHasher>(
     // convert processcontrol::Output into our Output
     runner.await?.map(|result| result.into())
 }
+
+async fn monitor_stream(name: &str, context: &str, stream: impl AsyncRead + Unpin) -> Result<()> {
+    let mut stream = BufReader::new(stream);
+    loop {
+        let mut buf = vec![];
```

Comment:
Does this allocate a new buffer every time through the loop? If so, maybe move this out of the loop and clear the buffer here.

Reply:
From discussion, leaving as is.